content/agent/kubernetes/event_collection.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Kubernetes Event Collection
+kind: documentation
+---
+
+Similarly to Agent 5, Agent 6 can collect events from the Kubernetes API server.
+Set the `collect_kubernetes_events` variable to `true` in the datadog.yaml. This can be achieved via the environment variable `DD_COLLECT_KUBERNETES_EVENTS` that is resolved at start time.
+You will need to give the Agent some rights to activate this feature: see the [RBAC][1]section.
+
+A [ConfigMap][3] can be used to store the `event.tokenKey` and the `event.tokenTimestamp`. It has to be deployed in the `default` namespace and be named `datadogtoken`.
+You can run `kubectl create configmap datadogtoken --from-literal="event.tokenKey"="0"` . You can also use the example in `manifests/datadog_configmap.yaml`.
+
+When the ConfigMap is used, if the Agent in charge (via the [Leader election][2]) of collecting the events dies, the next leader elected will use the ConfigMap to identify the last events pulled.
+This is in order to avoid duplicating the events collected, as well as to diminish stress on the API Server.
+
+## Leader Election
+
+Agent 6 supports a built-in leader election option for the Kubernetes event collector and the Kubernetes cluster-related checks (i.e. the Control Plane service check).
+
+This feature relies on Endpoints. You can enable it by setting the `DD_LEADER_ELECTION` environment variable to `true`. The Agent will need to have a set of actions allowed prior to its deployment; see the [RBAC][1] section for more details, and keep in mind that these RBAC entities will need to be created before the option is set.
```

Comment:
r/ `The Agent will need to have a set of actions allowed prior to its deployment` / `The Agent must have a set of actions allowed prior to its deployment` (...) r/`and keep in mind that these RBAC entities will need to be created before the option is set.` /`and keep in mind that these RBAC entities MUST be created before the option is set.`